html/Mogo/index.html

```diff
@@ -0,0 +1,1084 @@
+        <div>
+          <h4 class="single-offer__title">Photography</h4>
+          <p class="section-description single-offer__description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
+        </div>
+      </div>
+
+      <!-- Offer 2 -->
+      <div class="single-offer company-offer__single-offer">
+        <img class="single-offer__icon" src="./assests/images/company-offer/line-graph.svg" alt="">
+        <div>
+          <h4 class="single-offer__title">Web design</h4>
+          <p class="section-description single-offer__description">Ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
+        </div>
+      </div>
+
+      <!-- Offer 3 -->
+      <div class="single-offer company-offer__single-offer">
+        <img class="single-offer__icon" src="./assests/images/company-offer/computer-ok.svg" alt="">
+        <div>
+          <h4 class="single-offer__title">Creativity</h4>
+          <p class="section-description single-offer__description">Dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
+        </div>
+      </div>
+
+      <!-- Offer 4 -->
+      <div class="single-offer company-offer__single-offer">
+        <img class="single-offer__icon" src="./assests/images/company-offer/book-2.svg" alt="">
+        <div>
+          <h4 class="single-offer__title">SEO</h4>
+          <p class="section-description single-offer__description">Ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.</p>
+        </div>
+      </div>
+
+      <!-- Offer 5 -->
+      <div class="single-offer company-offer__single-offer">
+        <img class="single-offer__icon" src="./assests/images/company-offer/home.svg" alt="">
+        <div>
+          <h4 class="single-offer__title">Css/Html</h4>
+          <p class="section-description single-offer__description">Lorem dolor sit amet, consectetur adipiscing elit, sed do tempor.</p>
+        </div>
+      </div>
+
+      <!-- Offer 6 -->
+      <div class="single-offer company-offer__single-offer">
+        <img class="single-offer__icon" src="./assests/images/company-offer/image.svg" alt="">
+        <div>
+          <h4 class="single-offer__title">Digital</h4>
+          <p class="section-description single-offer__description">Sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
+        </div>
+      </div>
+
+    </div>
+  </section>
+
+  <!-- Unique design -->
+  <section class="unique-design">
+
+    <!-- Type -->
+    <p class="section-type section-type--black unique-design__type">
+      For all devices
+    </p>
+    
+    <!-- Title -->
+    <h2 class="section-title section-title--black unique-design__title">
+      Unique design
+    </h2>
+
+    <!-- Images -->
+    <div class="unique-design__container">
+      <!-- Tablet -->
+      <img class="unique-design__tablet" src="./assests/images/unique-design/display-bg-768-x-1024@2x.jpg" alt="Tablet view">
```

Comment:
alt not describe that you have in image and I'm just interested if there is other way to do it without two img tags?